bin/juttle-engine

```diff
@@ -7,7 +7,7 @@ var engine = require('..');
 
 var defaults = {
     'port': 8080,
-    'root': '/',
+    'root': process.cwd(),
```

Comment:
Another optionwould be to support multiple root paths somehow. This gets tricky with the directory listing of course.

Now we could always hard code support for "/examples" -- if it doesn't exist in the user's actual root directory, we could mash in the builtin examples.